main/src/main/java/cgeo/geocaching/unifiedmap/tileproviders/UserDefinedMapsforgeVTMOnlineSource.java

```diff
@@ -29,6 +29,7 @@ public class UserDefinedMapsforgeVTMOnlineSource extends AbstractMapsforgeVTMOnl
             }
             setTilePath(tilePath);
         }
+        supportsHillshading = true;
```

Comment:
some fine-tuning: some of those "supportsHilldhading = true;" are not needed anymore as the super class already sets this value